db/models.py

```diff
@@ -50,3 +54,69 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="orders"
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(
+        to=MovieSession, on_delete=models.CASCADE, related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order, on_delete=models.CASCADE, related_name="tickets"
+    )
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["movie_session", "row", "seat"], name="unique_ticket"
+            )
+        ]
+
+    def clean(self) -> None:
+        cinema_hall = self.movie_session.cinema_hall
+
+        if self.row > cinema_hall.rows:
+            raise ValidationError(
+                {
+                    "row": [
+                        f"row number must be in available range: (1, rows):"
+                        f" (1, {cinema_hall.rows})"
+                    ]
+                }
+            )
+
+        if self.seat > cinema_hall.seats_in_row:
+            raise ValidationError(
+                {
+                    "seat": [
+                        f"seat number must be in available range:"
+                        f" (1, seats_in_row):"
+                        f" (1, {cinema_hall.seats_in_row})"
+                    ]
+                }
+            )
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    def __str__(self) -> str:
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser` but does not add any additional fields or methods. Ensure this is intentional. If you plan to add custom fields or methods, consider implementing them here.